src/HW_2_OOP/RavenclawStudents.java

```diff
@@ -0,0 +1,37 @@
+package HW_2_OOP;
+
+public class RavenclawStudents extends HogwartsStudentGeneral {
```

Comment:
```suggestion
public class RavenclawStudent extends HogwartsStudentGeneral {
```
классы нужно называть в единственном числе